[2025-08-17T13:01:17.060+0000] {processor.py:186} INFO - Started process (PID=59) to work on /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:01:17.061+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-08-17T13:01:17.065+0000] {logging_mixin.py:190} INFO - [2025-08-17T13:01:17.065+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:01:18.349+0000] {logging_mixin.py:190} INFO - [2025-08-17T13:01:18.348+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-08-17T13:01:18.350+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:01:18.369+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 1.318 seconds
[2025-08-17T13:01:49.269+0000] {processor.py:186} INFO - Started process (PID=72) to work on /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:01:49.269+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-08-17T13:01:49.272+0000] {logging_mixin.py:190} INFO - [2025-08-17T13:01:49.272+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:01:49.914+0000] {logging_mixin.py:190} INFO - [2025-08-17T13:01:49.913+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-08-17T13:01:49.915+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:01:49.929+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.667 seconds
[2025-08-17T13:02:20.460+0000] {processor.py:186} INFO - Started process (PID=85) to work on /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:02:20.462+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-08-17T13:02:20.464+0000] {logging_mixin.py:190} INFO - [2025-08-17T13:02:20.464+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:02:21.110+0000] {logging_mixin.py:190} INFO - [2025-08-17T13:02:21.108+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-08-17T13:02:21.110+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:02:21.128+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.674 seconds
[2025-08-17T13:02:51.649+0000] {processor.py:186} INFO - Started process (PID=104) to work on /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:02:51.650+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-08-17T13:02:51.653+0000] {logging_mixin.py:190} INFO - [2025-08-17T13:02:51.652+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:02:52.284+0000] {logging_mixin.py:190} INFO - [2025-08-17T13:02:52.282+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-08-17T13:02:52.284+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:02:52.301+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.658 seconds
[2025-08-17T13:03:22.826+0000] {processor.py:186} INFO - Started process (PID=117) to work on /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:03:22.827+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-08-17T13:03:22.830+0000] {logging_mixin.py:190} INFO - [2025-08-17T13:03:22.830+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:03:23.454+0000] {logging_mixin.py:190} INFO - [2025-08-17T13:03:23.452+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-08-17T13:03:23.454+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:03:23.469+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.650 seconds
[2025-08-17T13:03:54.017+0000] {processor.py:186} INFO - Started process (PID=130) to work on /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:03:54.018+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-08-17T13:03:54.020+0000] {logging_mixin.py:190} INFO - [2025-08-17T13:03:54.020+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:03:54.691+0000] {logging_mixin.py:190} INFO - [2025-08-17T13:03:54.690+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-08-17T13:03:54.692+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:03:54.709+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.698 seconds
[2025-08-17T13:04:25.485+0000] {processor.py:186} INFO - Started process (PID=143) to work on /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:04:25.489+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-08-17T13:04:25.491+0000] {logging_mixin.py:190} INFO - [2025-08-17T13:04:25.491+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:04:25.978+0000] {logging_mixin.py:190} INFO - [2025-08-17T13:04:25.976+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-08-17T13:04:25.978+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:04:25.997+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.519 seconds
[2025-08-17T13:04:56.725+0000] {processor.py:186} INFO - Started process (PID=156) to work on /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:04:56.727+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-08-17T13:04:56.730+0000] {logging_mixin.py:190} INFO - [2025-08-17T13:04:56.730+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:04:57.196+0000] {logging_mixin.py:190} INFO - [2025-08-17T13:04:57.195+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-08-17T13:04:57.197+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:04:57.215+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.496 seconds
[2025-08-17T13:05:27.940+0000] {processor.py:186} INFO - Started process (PID=169) to work on /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:05:27.942+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-08-17T13:05:27.945+0000] {logging_mixin.py:190} INFO - [2025-08-17T13:05:27.945+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:05:28.418+0000] {logging_mixin.py:190} INFO - [2025-08-17T13:05:28.417+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-08-17T13:05:28.419+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:05:28.436+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.502 seconds
[2025-08-17T13:05:59.146+0000] {processor.py:186} INFO - Started process (PID=182) to work on /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:05:59.147+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-08-17T13:05:59.149+0000] {logging_mixin.py:190} INFO - [2025-08-17T13:05:59.149+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:05:59.604+0000] {logging_mixin.py:190} INFO - [2025-08-17T13:05:59.602+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-08-17T13:05:59.604+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:05:59.623+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.484 seconds
[2025-08-17T13:06:30.383+0000] {processor.py:186} INFO - Started process (PID=195) to work on /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:06:30.385+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-08-17T13:06:30.389+0000] {logging_mixin.py:190} INFO - [2025-08-17T13:06:30.389+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:06:30.867+0000] {logging_mixin.py:190} INFO - [2025-08-17T13:06:30.865+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-08-17T13:06:30.867+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:06:30.890+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.514 seconds
[2025-08-17T13:07:01.571+0000] {processor.py:186} INFO - Started process (PID=208) to work on /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:07:01.572+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-08-17T13:07:01.574+0000] {logging_mixin.py:190} INFO - [2025-08-17T13:07:01.574+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:07:02.036+0000] {logging_mixin.py:190} INFO - [2025-08-17T13:07:02.034+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-08-17T13:07:02.036+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:07:02.057+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.495 seconds
[2025-08-17T13:07:32.797+0000] {processor.py:186} INFO - Started process (PID=221) to work on /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:07:32.849+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-08-17T13:07:32.852+0000] {logging_mixin.py:190} INFO - [2025-08-17T13:07:32.852+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:07:33.320+0000] {logging_mixin.py:190} INFO - [2025-08-17T13:07:33.318+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-08-17T13:07:33.320+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:07:33.338+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.549 seconds
[2025-08-17T13:08:03.989+0000] {processor.py:186} INFO - Started process (PID=234) to work on /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:08:03.990+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-08-17T13:08:03.993+0000] {logging_mixin.py:190} INFO - [2025-08-17T13:08:03.993+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:08:04.483+0000] {logging_mixin.py:190} INFO - [2025-08-17T13:08:04.481+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-08-17T13:08:04.484+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:08:04.503+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.521 seconds
[2025-08-17T13:08:35.187+0000] {processor.py:186} INFO - Started process (PID=247) to work on /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:08:35.189+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-08-17T13:08:35.192+0000] {logging_mixin.py:190} INFO - [2025-08-17T13:08:35.192+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:08:35.668+0000] {logging_mixin.py:190} INFO - [2025-08-17T13:08:35.666+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-08-17T13:08:35.668+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:08:35.688+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.509 seconds
[2025-08-17T13:09:06.380+0000] {processor.py:186} INFO - Started process (PID=260) to work on /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:09:06.381+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-08-17T13:09:06.384+0000] {logging_mixin.py:190} INFO - [2025-08-17T13:09:06.383+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:09:06.859+0000] {logging_mixin.py:190} INFO - [2025-08-17T13:09:06.857+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-08-17T13:09:06.859+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:09:06.878+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.506 seconds
[2025-08-17T13:09:37.645+0000] {processor.py:186} INFO - Started process (PID=273) to work on /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:09:37.647+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-08-17T13:09:37.650+0000] {logging_mixin.py:190} INFO - [2025-08-17T13:09:37.649+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:09:38.122+0000] {logging_mixin.py:190} INFO - [2025-08-17T13:09:38.120+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-08-17T13:09:38.122+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:09:38.146+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.508 seconds
[2025-08-17T13:10:08.874+0000] {processor.py:186} INFO - Started process (PID=286) to work on /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:10:08.877+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-08-17T13:10:08.880+0000] {logging_mixin.py:190} INFO - [2025-08-17T13:10:08.879+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:10:09.344+0000] {logging_mixin.py:190} INFO - [2025-08-17T13:10:09.343+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-08-17T13:10:09.345+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:10:09.365+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.499 seconds
[2025-08-17T13:10:40.089+0000] {processor.py:186} INFO - Started process (PID=299) to work on /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:10:40.090+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-08-17T13:10:40.093+0000] {logging_mixin.py:190} INFO - [2025-08-17T13:10:40.093+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:10:40.567+0000] {logging_mixin.py:190} INFO - [2025-08-17T13:10:40.566+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-08-17T13:10:40.568+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:10:40.588+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.507 seconds
[2025-08-17T13:11:11.269+0000] {processor.py:186} INFO - Started process (PID=312) to work on /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:11:11.270+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-08-17T13:11:11.273+0000] {logging_mixin.py:190} INFO - [2025-08-17T13:11:11.273+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:11:11.757+0000] {logging_mixin.py:190} INFO - [2025-08-17T13:11:11.755+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-08-17T13:11:11.757+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:11:11.778+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.516 seconds
[2025-08-17T13:11:42.466+0000] {processor.py:186} INFO - Started process (PID=325) to work on /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:11:42.468+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-08-17T13:11:42.472+0000] {logging_mixin.py:190} INFO - [2025-08-17T13:11:42.471+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:11:42.957+0000] {logging_mixin.py:190} INFO - [2025-08-17T13:11:42.955+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-08-17T13:11:42.957+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:11:42.976+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.517 seconds
[2025-08-17T13:12:13.648+0000] {processor.py:186} INFO - Started process (PID=339) to work on /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:12:13.649+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-08-17T13:12:13.652+0000] {logging_mixin.py:190} INFO - [2025-08-17T13:12:13.651+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:12:14.108+0000] {logging_mixin.py:190} INFO - [2025-08-17T13:12:14.106+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-08-17T13:12:14.108+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:12:14.127+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.486 seconds
[2025-08-17T13:12:44.864+0000] {processor.py:186} INFO - Started process (PID=352) to work on /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:12:44.865+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-08-17T13:12:44.868+0000] {logging_mixin.py:190} INFO - [2025-08-17T13:12:44.868+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:12:45.330+0000] {logging_mixin.py:190} INFO - [2025-08-17T13:12:45.329+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-08-17T13:12:45.331+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:12:45.349+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.492 seconds
[2025-08-17T13:13:16.040+0000] {processor.py:186} INFO - Started process (PID=365) to work on /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:13:16.041+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-08-17T13:13:16.044+0000] {logging_mixin.py:190} INFO - [2025-08-17T13:13:16.044+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:13:16.518+0000] {logging_mixin.py:190} INFO - [2025-08-17T13:13:16.516+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-08-17T13:13:16.518+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:13:16.539+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.506 seconds
[2025-08-17T13:13:47.317+0000] {processor.py:186} INFO - Started process (PID=378) to work on /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:13:47.319+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-08-17T13:13:47.322+0000] {logging_mixin.py:190} INFO - [2025-08-17T13:13:47.321+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:13:47.790+0000] {logging_mixin.py:190} INFO - [2025-08-17T13:13:47.789+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-08-17T13:13:47.791+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:13:47.811+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.501 seconds
[2025-08-17T13:14:18.496+0000] {processor.py:186} INFO - Started process (PID=391) to work on /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:14:18.497+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-08-17T13:14:18.500+0000] {logging_mixin.py:190} INFO - [2025-08-17T13:14:18.500+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:14:18.962+0000] {logging_mixin.py:190} INFO - [2025-08-17T13:14:18.960+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-08-17T13:14:18.962+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:14:18.984+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.496 seconds
[2025-08-17T13:14:49.707+0000] {processor.py:186} INFO - Started process (PID=404) to work on /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:14:49.709+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-08-17T13:14:49.712+0000] {logging_mixin.py:190} INFO - [2025-08-17T13:14:49.712+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:14:50.172+0000] {logging_mixin.py:190} INFO - [2025-08-17T13:14:50.170+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-08-17T13:14:50.173+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:14:50.192+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.492 seconds
[2025-08-17T13:15:20.884+0000] {processor.py:186} INFO - Started process (PID=417) to work on /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:15:20.886+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-08-17T13:15:20.888+0000] {logging_mixin.py:190} INFO - [2025-08-17T13:15:20.888+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:15:21.347+0000] {logging_mixin.py:190} INFO - [2025-08-17T13:15:21.345+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-08-17T13:15:21.347+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:15:21.368+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.491 seconds
[2025-08-17T13:15:52.064+0000] {processor.py:186} INFO - Started process (PID=436) to work on /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:15:52.066+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-08-17T13:15:52.069+0000] {logging_mixin.py:190} INFO - [2025-08-17T13:15:52.069+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:15:52.543+0000] {logging_mixin.py:190} INFO - [2025-08-17T13:15:52.542+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-08-17T13:15:52.544+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:15:52.563+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.506 seconds
[2025-08-17T13:16:23.303+0000] {processor.py:186} INFO - Started process (PID=450) to work on /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:16:23.304+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-08-17T13:16:23.308+0000] {logging_mixin.py:190} INFO - [2025-08-17T13:16:23.307+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:16:23.803+0000] {logging_mixin.py:190} INFO - [2025-08-17T13:16:23.801+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-08-17T13:16:23.803+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:16:23.821+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.525 seconds
[2025-08-17T13:16:54.597+0000] {processor.py:186} INFO - Started process (PID=463) to work on /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:16:54.598+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-08-17T13:16:54.601+0000] {logging_mixin.py:190} INFO - [2025-08-17T13:16:54.601+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:16:55.080+0000] {logging_mixin.py:190} INFO - [2025-08-17T13:16:55.079+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-08-17T13:16:55.081+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:16:55.100+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.511 seconds
[2025-08-17T13:17:25.827+0000] {processor.py:186} INFO - Started process (PID=476) to work on /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:17:25.829+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-08-17T13:17:25.832+0000] {logging_mixin.py:190} INFO - [2025-08-17T13:17:25.832+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:17:26.290+0000] {logging_mixin.py:190} INFO - [2025-08-17T13:17:26.289+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-08-17T13:17:26.291+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:17:26.310+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.489 seconds
[2025-08-17T13:17:57.005+0000] {processor.py:186} INFO - Started process (PID=489) to work on /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:17:57.006+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-08-17T13:17:57.009+0000] {logging_mixin.py:190} INFO - [2025-08-17T13:17:57.009+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:17:57.477+0000] {logging_mixin.py:190} INFO - [2025-08-17T13:17:57.476+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-08-17T13:17:57.478+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:17:57.499+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.501 seconds
[2025-08-17T13:18:28.209+0000] {processor.py:186} INFO - Started process (PID=502) to work on /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:18:28.211+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-08-17T13:18:28.214+0000] {logging_mixin.py:190} INFO - [2025-08-17T13:18:28.213+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:18:28.676+0000] {logging_mixin.py:190} INFO - [2025-08-17T13:18:28.675+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-08-17T13:18:28.677+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:18:28.696+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.494 seconds
[2025-08-17T13:18:59.456+0000] {processor.py:186} INFO - Started process (PID=515) to work on /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:18:59.457+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-08-17T13:18:59.460+0000] {logging_mixin.py:190} INFO - [2025-08-17T13:18:59.460+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:18:59.939+0000] {logging_mixin.py:190} INFO - [2025-08-17T13:18:59.937+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-08-17T13:18:59.939+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:18:59.960+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.511 seconds
[2025-08-17T13:19:30.702+0000] {processor.py:186} INFO - Started process (PID=528) to work on /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:19:30.704+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-08-17T13:19:30.707+0000] {logging_mixin.py:190} INFO - [2025-08-17T13:19:30.707+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:19:31.196+0000] {logging_mixin.py:190} INFO - [2025-08-17T13:19:31.194+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-08-17T13:19:31.197+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:19:31.215+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.521 seconds
[2025-08-17T13:20:01.877+0000] {processor.py:186} INFO - Started process (PID=541) to work on /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:20:01.878+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-08-17T13:20:01.882+0000] {logging_mixin.py:190} INFO - [2025-08-17T13:20:01.881+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:20:02.373+0000] {logging_mixin.py:190} INFO - [2025-08-17T13:20:02.371+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-08-17T13:20:02.373+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:20:02.398+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.528 seconds
[2025-08-17T13:20:33.084+0000] {processor.py:186} INFO - Started process (PID=554) to work on /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:20:33.087+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-08-17T13:20:33.090+0000] {logging_mixin.py:190} INFO - [2025-08-17T13:20:33.090+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:20:33.574+0000] {logging_mixin.py:190} INFO - [2025-08-17T13:20:33.573+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-08-17T13:20:33.575+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:20:33.595+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.518 seconds
[2025-08-17T13:21:04.369+0000] {processor.py:186} INFO - Started process (PID=567) to work on /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:21:04.370+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-08-17T13:21:04.372+0000] {logging_mixin.py:190} INFO - [2025-08-17T13:21:04.372+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:21:04.849+0000] {logging_mixin.py:190} INFO - [2025-08-17T13:21:04.847+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-08-17T13:21:04.849+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:21:04.866+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.505 seconds
[2025-08-17T13:21:35.587+0000] {processor.py:186} INFO - Started process (PID=580) to work on /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:21:35.588+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-08-17T13:21:35.592+0000] {logging_mixin.py:190} INFO - [2025-08-17T13:21:35.591+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:21:36.057+0000] {logging_mixin.py:190} INFO - [2025-08-17T13:21:36.056+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-08-17T13:21:36.058+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:21:36.079+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.499 seconds
[2025-08-17T13:22:06.771+0000] {processor.py:186} INFO - Started process (PID=593) to work on /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:22:06.772+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-08-17T13:22:06.775+0000] {logging_mixin.py:190} INFO - [2025-08-17T13:22:06.775+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:22:07.240+0000] {logging_mixin.py:190} INFO - [2025-08-17T13:22:07.238+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-08-17T13:22:07.241+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:22:07.260+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.497 seconds
[2025-08-17T13:22:38.005+0000] {processor.py:186} INFO - Started process (PID=606) to work on /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:22:38.006+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-08-17T13:22:38.010+0000] {logging_mixin.py:190} INFO - [2025-08-17T13:22:38.009+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:22:38.491+0000] {logging_mixin.py:190} INFO - [2025-08-17T13:22:38.490+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-08-17T13:22:38.492+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:22:38.513+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.515 seconds
[2025-08-17T13:23:09.283+0000] {processor.py:186} INFO - Started process (PID=619) to work on /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:23:09.284+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-08-17T13:23:09.287+0000] {logging_mixin.py:190} INFO - [2025-08-17T13:23:09.287+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:23:09.776+0000] {logging_mixin.py:190} INFO - [2025-08-17T13:23:09.775+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-08-17T13:23:09.777+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:23:09.797+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.522 seconds
[2025-08-17T13:23:40.482+0000] {processor.py:186} INFO - Started process (PID=632) to work on /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:23:40.484+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-08-17T13:23:40.487+0000] {logging_mixin.py:190} INFO - [2025-08-17T13:23:40.487+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:23:40.972+0000] {logging_mixin.py:190} INFO - [2025-08-17T13:23:40.971+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-08-17T13:23:40.973+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:23:40.994+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.519 seconds
[2025-08-17T13:24:11.798+0000] {processor.py:186} INFO - Started process (PID=645) to work on /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:24:11.799+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-08-17T13:24:11.802+0000] {logging_mixin.py:190} INFO - [2025-08-17T13:24:11.802+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:24:12.271+0000] {logging_mixin.py:190} INFO - [2025-08-17T13:24:12.269+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-08-17T13:24:12.271+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:24:12.291+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.500 seconds
[2025-08-17T13:24:43.018+0000] {processor.py:186} INFO - Started process (PID=658) to work on /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:24:43.020+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-08-17T13:24:43.023+0000] {logging_mixin.py:190} INFO - [2025-08-17T13:24:43.023+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:24:43.495+0000] {logging_mixin.py:190} INFO - [2025-08-17T13:24:43.494+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-08-17T13:24:43.496+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:24:43.514+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.503 seconds
[2025-08-17T13:25:14.231+0000] {processor.py:186} INFO - Started process (PID=671) to work on /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:25:14.232+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-08-17T13:25:14.235+0000] {logging_mixin.py:190} INFO - [2025-08-17T13:25:14.234+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:25:14.697+0000] {logging_mixin.py:190} INFO - [2025-08-17T13:25:14.696+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-08-17T13:25:14.698+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:25:14.718+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.494 seconds
[2025-08-17T13:25:45.443+0000] {processor.py:186} INFO - Started process (PID=684) to work on /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:25:45.449+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-08-17T13:25:45.453+0000] {logging_mixin.py:190} INFO - [2025-08-17T13:25:45.452+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:25:45.919+0000] {logging_mixin.py:190} INFO - [2025-08-17T13:25:45.917+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-08-17T13:25:45.920+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:25:45.938+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.502 seconds
[2025-08-17T13:26:16.625+0000] {processor.py:186} INFO - Started process (PID=697) to work on /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:26:16.626+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-08-17T13:26:16.629+0000] {logging_mixin.py:190} INFO - [2025-08-17T13:26:16.628+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:26:17.098+0000] {logging_mixin.py:190} INFO - [2025-08-17T13:26:17.097+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-08-17T13:26:17.099+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:26:17.119+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.501 seconds
[2025-08-17T13:26:47.833+0000] {processor.py:186} INFO - Started process (PID=710) to work on /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:26:47.835+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-08-17T13:26:47.839+0000] {logging_mixin.py:190} INFO - [2025-08-17T13:26:47.838+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:26:48.330+0000] {logging_mixin.py:190} INFO - [2025-08-17T13:26:48.329+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-08-17T13:26:48.331+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:26:48.350+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.526 seconds
[2025-08-17T13:27:19.019+0000] {processor.py:186} INFO - Started process (PID=723) to work on /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:27:19.021+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-08-17T13:27:19.024+0000] {logging_mixin.py:190} INFO - [2025-08-17T13:27:19.023+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:27:19.488+0000] {logging_mixin.py:190} INFO - [2025-08-17T13:27:19.486+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-08-17T13:27:19.488+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:27:19.510+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.498 seconds
[2025-08-17T13:27:50.242+0000] {processor.py:186} INFO - Started process (PID=736) to work on /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:27:50.243+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-08-17T13:27:50.247+0000] {logging_mixin.py:190} INFO - [2025-08-17T13:27:50.246+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:27:50.728+0000] {logging_mixin.py:190} INFO - [2025-08-17T13:27:50.727+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-08-17T13:27:50.729+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:27:50.750+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.516 seconds
[2025-08-17T13:28:21.416+0000] {processor.py:186} INFO - Started process (PID=749) to work on /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:28:21.417+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-08-17T13:28:21.420+0000] {logging_mixin.py:190} INFO - [2025-08-17T13:28:21.419+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:28:21.919+0000] {logging_mixin.py:190} INFO - [2025-08-17T13:28:21.917+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-08-17T13:28:21.920+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:28:21.945+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.536 seconds
[2025-08-17T13:28:52.601+0000] {processor.py:186} INFO - Started process (PID=762) to work on /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:28:52.602+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-08-17T13:28:52.605+0000] {logging_mixin.py:190} INFO - [2025-08-17T13:28:52.605+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:28:53.087+0000] {logging_mixin.py:190} INFO - [2025-08-17T13:28:53.085+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-08-17T13:28:53.087+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:28:53.107+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.514 seconds
[2025-08-17T13:29:23.790+0000] {processor.py:186} INFO - Started process (PID=781) to work on /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:29:23.791+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-08-17T13:29:23.794+0000] {logging_mixin.py:190} INFO - [2025-08-17T13:29:23.794+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:29:24.263+0000] {logging_mixin.py:190} INFO - [2025-08-17T13:29:24.262+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-08-17T13:29:24.264+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:29:24.283+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.502 seconds
[2025-08-17T13:29:54.972+0000] {processor.py:186} INFO - Started process (PID=794) to work on /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:29:54.974+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-08-17T13:29:54.977+0000] {logging_mixin.py:190} INFO - [2025-08-17T13:29:54.976+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:29:55.466+0000] {logging_mixin.py:190} INFO - [2025-08-17T13:29:55.464+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-08-17T13:29:55.467+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:29:55.488+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.523 seconds
[2025-08-17T13:30:26.151+0000] {processor.py:186} INFO - Started process (PID=807) to work on /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:30:26.152+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-08-17T13:30:26.155+0000] {logging_mixin.py:190} INFO - [2025-08-17T13:30:26.155+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:30:26.642+0000] {logging_mixin.py:190} INFO - [2025-08-17T13:30:26.640+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-08-17T13:30:26.642+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:30:26.662+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.519 seconds
[2025-08-17T13:30:57.335+0000] {processor.py:186} INFO - Started process (PID=820) to work on /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:30:57.337+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-08-17T13:30:57.340+0000] {logging_mixin.py:190} INFO - [2025-08-17T13:30:57.340+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:30:57.819+0000] {logging_mixin.py:190} INFO - [2025-08-17T13:30:57.818+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-08-17T13:30:57.820+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:30:57.839+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.512 seconds
[2025-08-17T13:31:28.531+0000] {processor.py:186} INFO - Started process (PID=833) to work on /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:31:28.532+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-08-17T13:31:28.535+0000] {logging_mixin.py:190} INFO - [2025-08-17T13:31:28.534+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:31:29.000+0000] {logging_mixin.py:190} INFO - [2025-08-17T13:31:28.999+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-08-17T13:31:29.001+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:31:29.019+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.496 seconds
[2025-08-17T13:31:59.763+0000] {processor.py:186} INFO - Started process (PID=847) to work on /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:31:59.804+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-08-17T13:31:59.808+0000] {logging_mixin.py:190} INFO - [2025-08-17T13:31:59.807+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:32:00.292+0000] {logging_mixin.py:190} INFO - [2025-08-17T13:32:00.290+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-08-17T13:32:00.293+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:32:00.311+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.554 seconds
[2025-08-17T13:32:30.980+0000] {processor.py:186} INFO - Started process (PID=860) to work on /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:32:30.981+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-08-17T13:32:30.984+0000] {logging_mixin.py:190} INFO - [2025-08-17T13:32:30.984+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:32:31.463+0000] {logging_mixin.py:190} INFO - [2025-08-17T13:32:31.461+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-08-17T13:32:31.464+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:32:31.483+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.511 seconds
[2025-08-17T13:33:02.216+0000] {processor.py:186} INFO - Started process (PID=873) to work on /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:33:02.219+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-08-17T13:33:02.222+0000] {logging_mixin.py:190} INFO - [2025-08-17T13:33:02.222+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:33:02.722+0000] {logging_mixin.py:190} INFO - [2025-08-17T13:33:02.720+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-08-17T13:33:02.722+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:33:02.741+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.532 seconds
[2025-08-17T13:33:33.397+0000] {processor.py:186} INFO - Started process (PID=886) to work on /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:33:33.398+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-08-17T13:33:33.402+0000] {logging_mixin.py:190} INFO - [2025-08-17T13:33:33.401+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:33:33.873+0000] {logging_mixin.py:190} INFO - [2025-08-17T13:33:33.871+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-08-17T13:33:33.873+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:33:33.893+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.504 seconds
[2025-08-17T13:34:04.650+0000] {processor.py:186} INFO - Started process (PID=899) to work on /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:34:04.652+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-08-17T13:34:04.655+0000] {logging_mixin.py:190} INFO - [2025-08-17T13:34:04.654+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:34:05.130+0000] {logging_mixin.py:190} INFO - [2025-08-17T13:34:05.128+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-08-17T13:34:05.130+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:34:05.152+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.509 seconds
[2025-08-17T13:34:35.862+0000] {processor.py:186} INFO - Started process (PID=912) to work on /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:34:35.864+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-08-17T13:34:35.868+0000] {logging_mixin.py:190} INFO - [2025-08-17T13:34:35.867+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:34:36.364+0000] {logging_mixin.py:190} INFO - [2025-08-17T13:34:36.362+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-08-17T13:34:36.365+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:34:36.389+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.535 seconds
[2025-08-17T13:35:07.066+0000] {processor.py:186} INFO - Started process (PID=925) to work on /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:35:07.068+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-08-17T13:35:07.071+0000] {logging_mixin.py:190} INFO - [2025-08-17T13:35:07.071+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:35:07.556+0000] {logging_mixin.py:190} INFO - [2025-08-17T13:35:07.554+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-08-17T13:35:07.556+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:35:07.577+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.519 seconds
[2025-08-17T13:35:38.360+0000] {processor.py:186} INFO - Started process (PID=938) to work on /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:35:38.361+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-08-17T13:35:38.364+0000] {logging_mixin.py:190} INFO - [2025-08-17T13:35:38.363+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:35:38.849+0000] {logging_mixin.py:190} INFO - [2025-08-17T13:35:38.847+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-08-17T13:35:38.850+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:35:38.873+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.521 seconds
[2025-08-17T13:36:09.628+0000] {processor.py:186} INFO - Started process (PID=951) to work on /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:36:09.629+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-08-17T13:36:09.633+0000] {logging_mixin.py:190} INFO - [2025-08-17T13:36:09.632+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:36:10.095+0000] {logging_mixin.py:190} INFO - [2025-08-17T13:36:10.094+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-08-17T13:36:10.096+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:36:10.114+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.494 seconds
[2025-08-17T13:36:40.850+0000] {processor.py:186} INFO - Started process (PID=964) to work on /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:36:40.851+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-08-17T13:36:40.854+0000] {logging_mixin.py:190} INFO - [2025-08-17T13:36:40.854+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:36:41.343+0000] {logging_mixin.py:190} INFO - [2025-08-17T13:36:41.341+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-08-17T13:36:41.343+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:36:41.368+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.525 seconds
[2025-08-17T13:37:12.040+0000] {processor.py:186} INFO - Started process (PID=977) to work on /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:37:12.042+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-08-17T13:37:12.045+0000] {logging_mixin.py:190} INFO - [2025-08-17T13:37:12.044+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:37:12.532+0000] {logging_mixin.py:190} INFO - [2025-08-17T13:37:12.531+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-08-17T13:37:12.533+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:37:12.558+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.526 seconds
[2025-08-17T13:37:43.248+0000] {processor.py:186} INFO - Started process (PID=990) to work on /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:37:43.249+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-08-17T13:37:43.252+0000] {logging_mixin.py:190} INFO - [2025-08-17T13:37:43.252+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:37:43.737+0000] {logging_mixin.py:190} INFO - [2025-08-17T13:37:43.735+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-08-17T13:37:43.738+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:37:43.760+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.519 seconds
[2025-08-17T13:38:14.536+0000] {processor.py:186} INFO - Started process (PID=1004) to work on /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:38:14.538+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-08-17T13:38:14.541+0000] {logging_mixin.py:190} INFO - [2025-08-17T13:38:14.540+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:38:15.027+0000] {logging_mixin.py:190} INFO - [2025-08-17T13:38:15.025+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-08-17T13:38:15.028+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:38:15.049+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.520 seconds
[2025-08-17T13:38:45.737+0000] {processor.py:186} INFO - Started process (PID=1017) to work on /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:38:45.738+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-08-17T13:38:45.741+0000] {logging_mixin.py:190} INFO - [2025-08-17T13:38:45.741+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:38:46.231+0000] {logging_mixin.py:190} INFO - [2025-08-17T13:38:46.229+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-08-17T13:38:46.231+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:38:46.251+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.522 seconds
[2025-08-17T13:39:16.933+0000] {processor.py:186} INFO - Started process (PID=1030) to work on /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:39:16.934+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-08-17T13:39:16.938+0000] {logging_mixin.py:190} INFO - [2025-08-17T13:39:16.937+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:39:17.404+0000] {logging_mixin.py:190} INFO - [2025-08-17T13:39:17.403+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-08-17T13:39:17.405+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:39:17.424+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.499 seconds
[2025-08-17T13:39:48.151+0000] {processor.py:186} INFO - Started process (PID=1043) to work on /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:39:48.153+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-08-17T13:39:48.156+0000] {logging_mixin.py:190} INFO - [2025-08-17T13:39:48.155+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:39:48.622+0000] {logging_mixin.py:190} INFO - [2025-08-17T13:39:48.620+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-08-17T13:39:48.623+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:39:48.642+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.499 seconds
[2025-08-17T13:40:19.410+0000] {processor.py:186} INFO - Started process (PID=1056) to work on /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:40:19.411+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-08-17T13:40:19.414+0000] {logging_mixin.py:190} INFO - [2025-08-17T13:40:19.414+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:40:19.895+0000] {logging_mixin.py:190} INFO - [2025-08-17T13:40:19.893+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-08-17T13:40:19.896+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:40:19.916+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.513 seconds
[2025-08-17T13:40:50.608+0000] {processor.py:186} INFO - Started process (PID=1069) to work on /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:40:50.610+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-08-17T13:40:50.613+0000] {logging_mixin.py:190} INFO - [2025-08-17T13:40:50.613+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:40:51.100+0000] {logging_mixin.py:190} INFO - [2025-08-17T13:40:51.098+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-08-17T13:40:51.100+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:40:51.120+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.519 seconds
[2025-08-17T13:41:21.895+0000] {processor.py:186} INFO - Started process (PID=1082) to work on /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:41:21.896+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-08-17T13:41:21.899+0000] {logging_mixin.py:190} INFO - [2025-08-17T13:41:21.899+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:41:22.364+0000] {logging_mixin.py:190} INFO - [2025-08-17T13:41:22.363+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-08-17T13:41:22.365+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:41:22.384+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.496 seconds
[2025-08-17T13:41:53.119+0000] {processor.py:186} INFO - Started process (PID=1095) to work on /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:41:53.120+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-08-17T13:41:53.123+0000] {logging_mixin.py:190} INFO - [2025-08-17T13:41:53.123+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:41:53.590+0000] {logging_mixin.py:190} INFO - [2025-08-17T13:41:53.588+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-08-17T13:41:53.590+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:41:53.609+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.497 seconds
[2025-08-17T13:42:24.299+0000] {processor.py:186} INFO - Started process (PID=1108) to work on /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:42:24.300+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-08-17T13:42:24.303+0000] {logging_mixin.py:190} INFO - [2025-08-17T13:42:24.302+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:42:24.790+0000] {logging_mixin.py:190} INFO - [2025-08-17T13:42:24.788+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-08-17T13:42:24.790+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:42:24.813+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.522 seconds
[2025-08-17T13:42:55.486+0000] {processor.py:186} INFO - Started process (PID=1127) to work on /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:42:55.487+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-08-17T13:42:55.490+0000] {logging_mixin.py:190} INFO - [2025-08-17T13:42:55.490+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:42:55.979+0000] {logging_mixin.py:190} INFO - [2025-08-17T13:42:55.977+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-08-17T13:42:55.979+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:42:55.999+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.521 seconds
[2025-08-17T13:43:26.665+0000] {processor.py:186} INFO - Started process (PID=1140) to work on /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:43:26.666+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-08-17T13:43:26.669+0000] {logging_mixin.py:190} INFO - [2025-08-17T13:43:26.669+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:43:27.148+0000] {logging_mixin.py:190} INFO - [2025-08-17T13:43:27.147+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-08-17T13:43:27.149+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:43:27.169+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.512 seconds
[2025-08-17T13:43:57.851+0000] {processor.py:186} INFO - Started process (PID=1153) to work on /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:43:57.853+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-08-17T13:43:57.857+0000] {logging_mixin.py:190} INFO - [2025-08-17T13:43:57.856+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:43:58.349+0000] {logging_mixin.py:190} INFO - [2025-08-17T13:43:58.347+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-08-17T13:43:58.349+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:43:58.373+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.531 seconds
[2025-08-17T13:44:29.038+0000] {processor.py:186} INFO - Started process (PID=1167) to work on /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:44:29.040+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-08-17T13:44:29.043+0000] {logging_mixin.py:190} INFO - [2025-08-17T13:44:29.042+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:44:29.505+0000] {logging_mixin.py:190} INFO - [2025-08-17T13:44:29.503+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-08-17T13:44:29.505+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:44:29.524+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.493 seconds
[2025-08-17T13:45:00.290+0000] {processor.py:186} INFO - Started process (PID=1180) to work on /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:45:00.292+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-08-17T13:45:00.295+0000] {logging_mixin.py:190} INFO - [2025-08-17T13:45:00.294+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:45:00.773+0000] {logging_mixin.py:190} INFO - [2025-08-17T13:45:00.772+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-08-17T13:45:00.774+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:45:00.796+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.513 seconds
[2025-08-17T13:45:31.471+0000] {processor.py:186} INFO - Started process (PID=1193) to work on /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:45:31.473+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-08-17T13:45:31.476+0000] {logging_mixin.py:190} INFO - [2025-08-17T13:45:31.475+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:45:31.968+0000] {logging_mixin.py:190} INFO - [2025-08-17T13:45:31.967+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-08-17T13:45:31.969+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:45:31.988+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.526 seconds
[2025-08-17T13:46:02.655+0000] {processor.py:186} INFO - Started process (PID=1206) to work on /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:46:02.657+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-08-17T13:46:02.660+0000] {logging_mixin.py:190} INFO - [2025-08-17T13:46:02.659+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:46:03.151+0000] {logging_mixin.py:190} INFO - [2025-08-17T13:46:03.149+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-08-17T13:46:03.152+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:46:03.178+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.530 seconds
[2025-08-17T13:46:33.851+0000] {processor.py:186} INFO - Started process (PID=1219) to work on /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:46:33.852+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-08-17T13:46:33.856+0000] {logging_mixin.py:190} INFO - [2025-08-17T13:46:33.856+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:46:34.360+0000] {logging_mixin.py:190} INFO - [2025-08-17T13:46:34.358+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-08-17T13:46:34.360+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:46:34.380+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.536 seconds
[2025-08-17T13:47:05.125+0000] {processor.py:186} INFO - Started process (PID=1232) to work on /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:47:05.127+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-08-17T13:47:05.130+0000] {logging_mixin.py:190} INFO - [2025-08-17T13:47:05.130+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:47:05.599+0000] {logging_mixin.py:190} INFO - [2025-08-17T13:47:05.597+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-08-17T13:47:05.599+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:47:05.622+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.504 seconds
[2025-08-17T13:47:36.309+0000] {processor.py:186} INFO - Started process (PID=1246) to work on /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:47:36.309+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-08-17T13:47:36.312+0000] {logging_mixin.py:190} INFO - [2025-08-17T13:47:36.312+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:47:36.773+0000] {logging_mixin.py:190} INFO - [2025-08-17T13:47:36.771+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-08-17T13:47:36.773+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:47:36.791+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.490 seconds
[2025-08-17T13:48:07.562+0000] {processor.py:186} INFO - Started process (PID=1259) to work on /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:48:07.563+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-08-17T13:48:07.566+0000] {logging_mixin.py:190} INFO - [2025-08-17T13:48:07.566+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:48:08.047+0000] {logging_mixin.py:190} INFO - [2025-08-17T13:48:08.045+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-08-17T13:48:08.047+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:48:08.065+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.511 seconds
[2025-08-17T13:48:38.789+0000] {processor.py:186} INFO - Started process (PID=1272) to work on /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:48:38.790+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-08-17T13:48:38.793+0000] {logging_mixin.py:190} INFO - [2025-08-17T13:48:38.792+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:48:39.277+0000] {logging_mixin.py:190} INFO - [2025-08-17T13:48:39.276+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-08-17T13:48:39.278+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:48:39.297+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.516 seconds
[2025-08-17T13:49:09.963+0000] {processor.py:186} INFO - Started process (PID=1285) to work on /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:49:09.965+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-08-17T13:49:09.968+0000] {logging_mixin.py:190} INFO - [2025-08-17T13:49:09.967+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:49:10.465+0000] {logging_mixin.py:190} INFO - [2025-08-17T13:49:10.464+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-08-17T13:49:10.466+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:49:10.484+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.529 seconds
[2025-08-17T13:49:41.166+0000] {processor.py:186} INFO - Started process (PID=1298) to work on /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:49:41.168+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-08-17T13:49:41.171+0000] {logging_mixin.py:190} INFO - [2025-08-17T13:49:41.170+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:49:41.648+0000] {logging_mixin.py:190} INFO - [2025-08-17T13:49:41.647+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-08-17T13:49:41.649+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:49:41.671+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.512 seconds
[2025-08-17T13:50:12.370+0000] {processor.py:186} INFO - Started process (PID=1311) to work on /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:50:12.371+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-08-17T13:50:12.374+0000] {logging_mixin.py:190} INFO - [2025-08-17T13:50:12.374+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:50:12.858+0000] {logging_mixin.py:190} INFO - [2025-08-17T13:50:12.857+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-08-17T13:50:12.859+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:50:12.876+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.513 seconds
[2025-08-17T13:50:43.582+0000] {processor.py:186} INFO - Started process (PID=1324) to work on /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:50:43.584+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-08-17T13:50:43.587+0000] {logging_mixin.py:190} INFO - [2025-08-17T13:50:43.586+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:50:44.049+0000] {logging_mixin.py:190} INFO - [2025-08-17T13:50:44.048+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-08-17T13:50:44.050+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:50:44.070+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.495 seconds
[2025-08-17T13:51:14.806+0000] {processor.py:186} INFO - Started process (PID=1337) to work on /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:51:14.807+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-08-17T13:51:14.810+0000] {logging_mixin.py:190} INFO - [2025-08-17T13:51:14.809+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:51:15.271+0000] {logging_mixin.py:190} INFO - [2025-08-17T13:51:15.270+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-08-17T13:51:15.272+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:51:15.291+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.492 seconds
[2025-08-17T13:51:45.989+0000] {processor.py:186} INFO - Started process (PID=1350) to work on /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:51:45.991+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-08-17T13:51:45.994+0000] {logging_mixin.py:190} INFO - [2025-08-17T13:51:45.994+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:51:46.459+0000] {logging_mixin.py:190} INFO - [2025-08-17T13:51:46.458+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-08-17T13:51:46.460+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:51:46.479+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.497 seconds
[2025-08-17T13:52:17.193+0000] {processor.py:186} INFO - Started process (PID=1363) to work on /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:52:17.194+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-08-17T13:52:17.196+0000] {logging_mixin.py:190} INFO - [2025-08-17T13:52:17.196+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:52:17.656+0000] {logging_mixin.py:190} INFO - [2025-08-17T13:52:17.655+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-08-17T13:52:17.657+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:52:17.674+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.489 seconds
[2025-08-17T13:52:48.392+0000] {processor.py:186} INFO - Started process (PID=1376) to work on /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:52:48.394+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-08-17T13:52:48.397+0000] {logging_mixin.py:190} INFO - [2025-08-17T13:52:48.396+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:52:48.858+0000] {logging_mixin.py:190} INFO - [2025-08-17T13:52:48.857+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-08-17T13:52:48.859+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:52:48.878+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.493 seconds
[2025-08-17T13:53:19.615+0000] {processor.py:186} INFO - Started process (PID=1389) to work on /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:53:19.616+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-08-17T13:53:19.619+0000] {logging_mixin.py:190} INFO - [2025-08-17T13:53:19.618+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:53:20.077+0000] {logging_mixin.py:190} INFO - [2025-08-17T13:53:20.076+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-08-17T13:53:20.078+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:53:20.097+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.489 seconds
[2025-08-17T13:53:50.818+0000] {processor.py:186} INFO - Started process (PID=1402) to work on /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:53:50.820+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-08-17T13:53:50.823+0000] {logging_mixin.py:190} INFO - [2025-08-17T13:53:50.823+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:53:51.310+0000] {logging_mixin.py:190} INFO - [2025-08-17T13:53:51.308+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-08-17T13:53:51.310+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:53:51.330+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.520 seconds
[2025-08-17T13:54:21.994+0000] {processor.py:186} INFO - Started process (PID=1415) to work on /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:54:21.995+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-08-17T13:54:21.998+0000] {logging_mixin.py:190} INFO - [2025-08-17T13:54:21.998+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:54:22.485+0000] {logging_mixin.py:190} INFO - [2025-08-17T13:54:22.483+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-08-17T13:54:22.485+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:54:22.505+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.518 seconds
[2025-08-17T13:54:53.196+0000] {processor.py:186} INFO - Started process (PID=1427) to work on /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:54:53.198+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-08-17T13:54:53.201+0000] {logging_mixin.py:190} INFO - [2025-08-17T13:54:53.200+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:54:53.678+0000] {logging_mixin.py:190} INFO - [2025-08-17T13:54:53.677+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-08-17T13:54:53.679+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:54:53.700+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.512 seconds
[2025-08-17T13:55:24.455+0000] {processor.py:186} INFO - Started process (PID=1440) to work on /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:55:24.457+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-08-17T13:55:24.460+0000] {logging_mixin.py:190} INFO - [2025-08-17T13:55:24.460+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:55:24.948+0000] {logging_mixin.py:190} INFO - [2025-08-17T13:55:24.946+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-08-17T13:55:24.948+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:55:24.972+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.524 seconds
[2025-08-17T13:55:55.652+0000] {processor.py:186} INFO - Started process (PID=1453) to work on /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:55:55.654+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-08-17T13:55:55.657+0000] {logging_mixin.py:190} INFO - [2025-08-17T13:55:55.656+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:55:56.136+0000] {logging_mixin.py:190} INFO - [2025-08-17T13:55:56.134+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-08-17T13:55:56.136+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:55:56.156+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.512 seconds
[2025-08-17T13:56:26.860+0000] {processor.py:186} INFO - Started process (PID=1472) to work on /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:56:26.861+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-08-17T13:56:26.864+0000] {logging_mixin.py:190} INFO - [2025-08-17T13:56:26.864+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:56:27.353+0000] {logging_mixin.py:190} INFO - [2025-08-17T13:56:27.351+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-08-17T13:56:27.354+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:56:27.374+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.522 seconds
[2025-08-17T13:56:57.417+0000] {processor.py:186} INFO - Started process (PID=1486) to work on /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:56:57.419+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-08-17T13:56:57.422+0000] {logging_mixin.py:190} INFO - [2025-08-17T13:56:57.421+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:56:57.899+0000] {logging_mixin.py:190} INFO - [2025-08-17T13:56:57.897+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-08-17T13:56:57.900+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:56:57.919+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.510 seconds
[2025-08-17T13:57:28.635+0000] {processor.py:186} INFO - Started process (PID=1499) to work on /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:57:28.636+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-08-17T13:57:28.639+0000] {logging_mixin.py:190} INFO - [2025-08-17T13:57:28.639+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:57:29.118+0000] {logging_mixin.py:190} INFO - [2025-08-17T13:57:29.116+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-08-17T13:57:29.118+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:57:29.140+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.512 seconds
[2025-08-17T13:57:59.902+0000] {processor.py:186} INFO - Started process (PID=1512) to work on /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:57:59.903+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-08-17T13:57:59.906+0000] {logging_mixin.py:190} INFO - [2025-08-17T13:57:59.906+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:58:00.399+0000] {logging_mixin.py:190} INFO - [2025-08-17T13:58:00.398+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-08-17T13:58:00.400+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:58:00.419+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.525 seconds
[2025-08-17T13:58:31.094+0000] {processor.py:186} INFO - Started process (PID=1525) to work on /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:58:31.095+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-08-17T13:58:31.098+0000] {logging_mixin.py:190} INFO - [2025-08-17T13:58:31.097+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:58:31.577+0000] {logging_mixin.py:190} INFO - [2025-08-17T13:58:31.575+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-08-17T13:58:31.578+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:58:31.602+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.515 seconds
[2025-08-17T13:59:02.279+0000] {processor.py:186} INFO - Started process (PID=1538) to work on /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:59:02.281+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-08-17T13:59:02.284+0000] {logging_mixin.py:190} INFO - [2025-08-17T13:59:02.284+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:59:02.773+0000] {logging_mixin.py:190} INFO - [2025-08-17T13:59:02.772+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-08-17T13:59:02.774+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:59:02.794+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.523 seconds
[2025-08-17T13:59:33.576+0000] {processor.py:186} INFO - Started process (PID=1551) to work on /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:59:33.577+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-08-17T13:59:33.581+0000] {logging_mixin.py:190} INFO - [2025-08-17T13:59:33.580+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:59:34.061+0000] {logging_mixin.py:190} INFO - [2025-08-17T13:59:34.060+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-08-17T13:59:34.062+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-08-17T13:59:34.084+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.517 seconds
